### PR TITLE
feat(adoption): post adoption support

### PR DESCRIPTION
**Pull request by AmandaKr**
## Jira issue

JIRA issue: [FSU-303](https://klau.atlassian.net/browse/FSU-303)

## Describe your changes
This PR introduces Post Adoption Support section.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/components/illustration/Illustration.jsx [hunk 1]
```diff
@@ -62,6 +66,10 @@ export const illustrations = {
   pet: Pet,
   standingOrder: StandingOrder,
   confirmation: Confirmation,
+  adoptionSupport: AdoptionSupport,
+  behaviorist: Behaviorist,
```
Comment (im-klau) [inline packages/design-system/components/illustration/Illustration.jsx:70]: duplicated illustration :D 

Zrzut ekranu 2023-07-24 o 11 24 39
Zrzut ekranu 2023-07-24 o 11 24 29

**Review by nata-kab — APPROVED** [review]
Great job, the code is clean and simple to understand 🚀 🤩

**Review by im-klau — APPROVED** [review]
Looks good! 🚀

**Review by paulgrym — APPROVED** [review]
Great job 👏

**Pull request merged**